src/Features/Onboarding.php

```diff
@@ -513,6 +513,7 @@ public static function get_allowed_plugins() {
 			array(
 				'facebook-for-woocommerce'            => 'facebook-for-woocommerce/facebook-for-woocommerce.php',
 				'mailchimp-for-woocommerce'           => 'mailchimp-for-woocommerce/mailchimp-woocommerce.php',
+				'google-ads'                          => 'woocommerce-google-ads/woocommerce-google-ads.php',
```

Comment:
This fails since this array looks for `google-ads` in the WP.org repo and will return a `Plugin not found` error.

I'll ping @mattyza and see if there's a way to do this via the WCCOM API.  We're also planning to move the Jetpack connection step later in the profiler in https://github.com/woocommerce/woocommerce-admin/issues/3709 which means if a connection is necessary, this won't work in the future.

Reply:
Historically, when adding plugins to the core onboarding wizard, a prerequisite was publishing the extensions on WP.org (see p7bbVw-2Oa-p2 for an example).

Reply:
Thanks for that link, @jeffstieler!  Uploading to the WP.org repo sounds like a much better path than trying to download via some WCCOM API.

@becdetat Let's put this PR on hold for now until we can get that plugin uploaded.

Reply:
the current plugin is a bit outdated but can be used until it’s updated?
https://wordpress.org/plugins/kliken-all-in-one-marketing/

Reply:
Unfortunately, that plugin won't meet partnership obligations so we'll need to wait for the official one.

Reply:
I think we can now replace this line with `kliken-marketing-for-google/kliken-marketing-for-google.php` and this PR should be ready for a final test and merge.